packages/engine/lib/error/src/frame.rs

```diff
@@ -169,10 +177,7 @@ impl FrameRepr {
         unsafe { (self.vtable.object_ref)(self) }
     }
 
-    fn downcast<C>(&self) -> Option<NonNull<C>>
-    where
-        C: Context,
-    {
+    fn downcast<C: Any>(&self) -> Option<NonNull<C>> {
         let target = TypeId::of::<C>();
         // Use vtable to attach C's native vtable for the right original type C.
```

Comment:
Can you add a safety by the way?

Reply:
I enforced this for this crate: 5df6e3a3a2029dd2e375a5fb5e7dec6b0ec49adf